src/TwigMiddleware.php

```diff
@@ -54,53 +42,31 @@ class TwigMiddleware implements MiddlewareInterface
      */
     public static function create(App $app, string $containerKey = 'view'): self
     {
-        if ($app->getContainer() === null) {
+        $container = $app->getContainer();
+        if ($container === null) {
             throw new RuntimeException('The app does not have a container.');
         }
 
-        $twig = $app->getContainer()->get($containerKey);
-        if (!($twig instanceof Twig)) {
-            throw new RuntimeException(sprintf('Twig could not be found in the container (key=%s).', $containerKey));
-        }
-
         return new self(
-            $twig,
-            $app->getContainer(),
+            $container->get($containerKey),
```

Comment:
Same thing here, need to use `$container->has(...)` before using the `get()` method

Reply:
I don't think this will do much good cause the traceback will make it very clearly where it gone wrong.

Reply:
Sorry, that’s not sufficient for me. I want us to handle that case locally and throw an appropriate runtime exception with a well formulated message.

Reply:
Okay

Reply:
Done